tox.ini

```
@@ -31,6 +31,11 @@ max-line-length = 160
 builtins = _
 exclude = .git,.tox
 
+[testenv:docs]
+deps = -r{toxinidir}/doc/requirements.txt
+commands =
+  sphinx-build -E -W -d doc/build/doctrees -b html doc/source/ doc/build/html
```

Maybe make it strict?
```suggestion
  sphinx-build -E -W --keep-going -n -d doc/build/doctrees -b html doc/source/ doc/build/html
```